src/include/sof/audio/component.h

```diff
@@ -559,6 +559,16 @@ struct comp_ipc_config {
 #endif
 };
 
+/**
+ * \enum module_process_type
+ * \brief Module-proces type
+ */
+enum module_process_type {
+	MODULE_PROCESS_STREAM,
```

Comment:
Can we have some inline comments describing each type with examples and how to categorize. Can be in another PR .

Reply:
yes, I removed this patch, only keep 3 patches for this PR, and will submit new PR later.